content/python/concepts/time-module/sleep.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.sleep()'
+Description: 'The function suspends execution of current thread for a given time.'
+Subject: 'Computer Science'
+Tags: 
+ -'Functions'
+ -'Time'
+CatalogContent:
+ -'learn-python'
+ - 'concepts/time-module'
+---
+
+### **.sleep()**
```

Comment:
Remove this header. The metadata provides the title for the entry.

Reply:
Removed the header.